core/timer.go

```diff
@@ -0,0 +1,77 @@
+package core
+
+import (
+	"sync"
+	"time"
+
+	"github.com/benbjohnson/clock"
+)
+
+const wakeupWaitTime = 30 * time.Second
```

Comment:
sollten wir das in timerduration umbenennen oder sogar beim NewTimer setzen so das wir ihn für andere Sachen wieder verwenden können?

Reply:
wakeupTimeout

wäre konsistent, fürs erste reicht const